### DISCUSSION TITLE
Assistance Needed with URL Structure on One.com Hosted Website

### DISCUSSION THREAD

**Discussion by mogumogudj** [vercel/community#6380] [Help] [answered]
### URL Structure on One.com Hosted Website with Vercel

Dear Vercel Community,

I'm reaching out for some guidance regarding the URL structure of my website hosted on Vercel. Currently, I have two domains, wolfverelst.be and wolfverelst.com, both connected to my one.com-hosted website via DNS records with Vercel. The site itself is built from scratch using raw HTML and CSS.

While everything is functioning correctly, I've noticed that when navigating to different HTML pages, the URL displays the .html extension (e.g., www.wolfverelst.be/projects.html). I would prefer to remove this extension for a cleaner URL structure, like "[wolfverelst.com/projects](http://www.wolfverelst.com/projects)".

Could anyone advise me on how to achieve this for all HTML pages on my website? Alternatively, if this is something that the Vercel team can assist with, I would greatly appreciate your support in implementing the necessary changes. Or is this something in my (newbie) code structure?

Thank you all for your attention to this matter. I'm looking forward to your insights and suggestions.

Warm regards,
Wolf

### Example

https://www.wolfverelst.com/

### Steps to Reproduce

/

**Comment by amyegan** [comment, marked as answer]
Hey @mogumogudj. This doesn't look like a problem with your code structure. If you add a `vercel.json` configuration file with [cleanUrls](https://vercel.com/docs/projects/project-configuration#cleanurls) set to `true` then the file extension won't be in the URL.

Please give that a try and let me know if you run into any trouble with it! 🙂

**Reply by mogumogudj** [reply]
Yes, this finally works for some reason, thank you very much
